### PR TITLE
Add baloise.ch

### PR DESCRIPTION
**Pull request by kisst**

### CODE REVIEW

File: entries/b/baloise.ch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Baloise Insurance": {
+    "domain": "baloise.ch",
+    "img": "baloise.ch.png",
```
Comment (phallobst) [inline entries/b/baloise.ch.json]: This is the logo in SVG format, which we prefer:
baloise ch

File: entries/b/baloise.ch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "tfa": [
+      "sms"
+    ],
+    "keywords": [
+      "other"
```
Comment (phallobst) [inline entries/b/baloise.ch.json]:
```suggestion
      "finance"
```

**Review by phallobst — CHANGES_REQUESTED** [review]
Is there some publicly accessible documentation? If not, can you provide a screenshot of the setup process (with your personal information redacted)?

**Comment by kisst** [comment]
> Is there some publicly accessible documentation? If not, can you provide a screenshot of the setup process (with your personal information redacted)?

Not really, neither easy to produce screenshots at this point as it's part of the registration, it's not optional it's mandatory.

**Comment by Carlgo11** [comment]
We need some sort of screenshots to confirm. Either of the setup process, login process or settings page.
Also, when are SMS codes required? On login attempts or just for certain actions? Is a password also required?

__Remember to blur or crop out any personal information linked to you when uploading the screenshot(s).__

**Comment by kisst** [comment]
SMS is sent out for each login and the sessions are short lived, the registration is already impossible without a phone number.

https://www.baloise.ch/en/private-customers/contact-services/customer-portal-mybaloise/registration/page/signup/step0

screenshoot

**Comment by Carlgo11** [comment]
And a password is also required?

**Comment by kisst** [comment]
screenshoot_step1
Yes step 1 is U/P and step 2 is an SMS with a one time password

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**